### PR TITLE
feat(mixpanel): add alias on sdk load

### PR DESCRIPTION
**Pull request by Gauravudia**
## PR Description

Adding alias call on sdk load for mixpanel with a flag `sendAliasOnLoad` 

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: src/core/analytics.js [hunk 1]
```diff
@@ -352,7 +352,7 @@ class Analytics {
               self.failedToBeLoadedIntegration.push(intgInstance);
             }
           }
-        }, 100);
+        }, 1);
```
Comment (bardisg) [inline src/core/analytics.js:355]: this is too intensive I belive in terms of cpu consumeption imagine 8 destinations that take 300ms to load. how may times will you invoke this method?

File: src/integrations/Mixpanel/browser.js [hunk 2]
```diff
@@ -23,6 +23,7 @@
 /* eslint-disable no-underscore-dangle */
 /* eslint-disable class-methods-use-this */
 import get from 'get-value';
+import isEmpty from 'lodash.isempty';
```
Comment (bardisg) [inline src/integrations/Mixpanel/browser.js:26]: why add a new lodash method? cant we just use !isNotEmpty instead?

File: .size-limit.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ module.exports = [
     name: 'Core - CDN',
     path: 'dist/legacy/rudder-analytics.min.js',
     gzip: true,
-    limit: '37.2 kB',
+    limit: '40.61 kB',
```
Comment (bardisg) [inline .size-limit.js:10]: this is too much! what have we added?

**Review by bardisg — CHANGES_REQUESTED** [review]

File: .size-limit.js (hunk 3 above)
Comment (Gauravudia) [reply .size-limit.js]: only import was lodash.empty. `npm run check:size` is showing this size limit, Removed the lodash.empty import and reduced the size limit.

File: src/integrations/Mixpanel/browser.js (hunk 2 above)
Comment (Gauravudia) [reply src/integrations/Mixpanel/browser.js]: Removed the lodash import

**Pull request closed without merging**